examples_tests/01.HelloWorld/main.cpp

```diff
@@ -286,13 +302,236 @@ int main()
 	sc_params.queueFamilyIndices = queueFamilyIndices;
 	sc_params.imageSharingMode = imageSharingMode;
 	sc_params.preTransform = preTransform;
+	sc_params.imageUsage = asset::IImage::EUF_COLOR_ATTACHMENT_BIT;
 
 	core::smart_refctd_ptr<video::ISwapchain> swapchain = device->createSwapchain(std::move(sc_params));
 
-	while (true)
+	// Create render pass
+	video::IGPURenderpass::SCreationParams::SAttachmentDescription attachmentDescription;
+	attachmentDescription.format = surfaceFormat.format; // this should be same as the imageview used for this attachment
+	attachmentDescription.samples = asset::IImage::ESCF_1_BIT;
+	attachmentDescription.loadOp = video::IGPURenderpass::ELO_CLEAR; // when the first subpass begins with this attachment, clear its color and depth components
+	attachmentDescription.storeOp = video::IGPURenderpass::ESO_STORE; // when the last subpass ends with this attachment, store its results
+	attachmentDescription.initialLayout = asset::EIL_UNDEFINED;
+	attachmentDescription.finalLayout = asset::EIL_PRESENT_SRC_KHR;

```

Comment:
@Crisspl why does this enum have a KHR suffix? no need for our own engine enums to have these

Reply:
no reason, just taken names from vulkan and changed prefix. Can rename on will

Reply:
`KHR` is kinda redundant there, @achalpandeyy change please.

Reply:
Resolved